qcodes/utils/types.py

```diff
@@ -54,6 +54,7 @@
 """
 
 numpy_concrete_complex = (np.complex64, np.complex128)
+numpy_complex_map_size2type = {np.dtype(t).itemsize: t for t in numpy_concrete_complex}
```

Comment:
this is now not needed, after the changes to complex adapt and convert were removed, right?